src/arvcamera.c

```diff
@@ -2427,6 +2427,65 @@ arv_camera_is_binning_available (ArvCamera *camera, GError **error)
 	return horizontal && vertical;
 }
 
+/**
+ * arv_camera_get_representation_gain:
+ * @camera: a #ArvCamera
+ * @error: a #GError placeholder, %NULL to ignore
+ *
+ * Return: enum ArvGcRepresentation, -1 if not available.
+ *
+ * Since: 
+ */
+
+ArvGcRepresentation
+arv_camera_get_representation_gain (ArvCamera *camera, GError **error)
+{
+	ArvCameraPrivate *priv = arv_camera_get_instance_private (camera);
+
+	g_return_val_if_fail (ARV_IS_CAMERA (camera), FALSE);
+
+	if (priv->has_gain)
+		return arv_device_get_representation (priv->device, "Gain", error);
+
+	if (priv->gain_raw_as_float)
+		return arv_device_get_representation (priv->device, "GainRaw", error);
+	if (priv->gain_abs_as_float)
+		return arv_device_get_representation (priv->device, "GainAbs", error);
+
+	return arv_device_get_representation (priv->device, "GainRaw", error);
+}
+
+/**
+ * arv_camera_get_representation_exposure_time:
+ * @camera: a #ArvCamera
+ * @error: a #GError placeholder, %NULL to ignore
+ *
+ * Return: enum ArvGcRepresentation, -1 if not available.
+ *
+ * Since: 
+ */
+
+ArvGcRepresentation
+arv_camera_get_representation_exposure_time (ArvCamera *camera, GError **error)
```

Comment:
Rename this function to arv_camera_get_exposure_time_representation()
Remove the error placeholder

Reply:
updated in b9c1305b92d0322ec1c7f1ebef44a47d78db6a99